studio/components/to-be-cleaned/Integration/Vercel.utils.ts

```diff
@@ -48,7 +48,7 @@ export async function fetchVercelProject({
     url = `${url}?${query}`
   }
   const data = await get(url, {
-    headers: { Authorization: `Bearer ${vercelToken}` },
+    headers: { vercel_authorization: `Bearer ${vercelToken}` },
```

Comment:
might not matter but wondering if might be good to follow http headers naming convention
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

"Vercel-Authorization"

im guessing need to update API too if thats the case

Reply:
It's an unofficial header key so i treat it as a normal dictionary key. Should be ok for our use-case